core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCatalog.java

```diff
@@ -450,13 +427,12 @@ public void testDropNamespace() throws IOException {
     Lists.newArrayList(tbl1, tbl2)
         .forEach(t -> catalog.createTable(t, SCHEMA, PartitionSpec.unpartitioned()));
 
-    AssertHelpers.assertThrows(
-        "Should fail to drop namespace is not empty " + namespace1,
-        NamespaceNotEmptyException.class,
-        "Namespace " + namespace1 + " is not empty.",
-        () -> {
-          catalog.dropNamespace(Namespace.of("db"));
-        });
+    Assertions.assertThatThrownBy(
+            () -> {
```

Comment:
the {} can be removed here and further above

Reply:
thanks, fixed!